### DISCUSSION TITLE
Call for testers

### DISCUSSION THREAD

**Discussion by protoism** [gwt-plugins/gwt-eclipse-plugin#436] [General]
Hi all,
does anyone want to candidate as a release tester?
We need to try to reasonably cover the matrix

* Eclipse version
* O/S
* Java version

**Comment by protoism** [comment]
I candidate myself: Windows / Eclipse Java 2022-12 / JDK 17

**Reply by axeluhl** [reply]
I can test on Ubuntu 20.04 LTS / Eclipse for Eclipse Committers 2022-12

**Reply by protoism** [reply]
Thanks!

**Reply by protoism** [reply]
@axeluhl can you file your results here: https://github.com/gwt-plugins/gwt-eclipse-plugin/discussions/442

**Comment by eliasbalasis** [comment]
I could test with Eclipse 2022-12 on Windows 10 64bit and Java8

**Reply by protoism** [reply]
@eliasbalasis can you file your results here: https://github.com/gwt-plugins/gwt-eclipse-plugin/discussions/442

**Comment by kazuohprc** [comment]
![image](https://user-images.githubusercontent.com/90484678/216030101-a9a32cde-2c43-46a9-9de1-020d3b07e567.png)
windows10 64bit eclipse 2022-12 jdk1.8
It worked.
I changed to jdk17 and tested it, but it didn't work.
When I returned to jdk1.8 again, it did not work and I investigated the cause
I couldn't change it with a plugin in GWT2.7.

**Reply by protoism** [reply]
Hi @kazuohprc, 
this place, probably it was not clear, is for people who intend to test all releases, say... 4.0.0.. 4.1.0... and so on.
Can you file an issue for your problem, possibly with more information, such as... logs, snapshots, any hint about what is failing?
BTW, I'm quite surprised that 2022-12 is working with jdk 1.8, as it should require at least 2022-12

**Reply by kazuohprc** [reply]
thank you for your reply.
The problem this time is that you can't change GWTSDK from 2.9 to 2.7 with GWT pulgin.
Perhaps there is a problem in the process of applying SDK changes within the plugin,
In the image, 2.9 is displayed, but change this to 2.7, apply and close, but when you open it again, it remains 2.9.
![image](https://user-images.githubusercontent.com/90484678/216175317-48296916-81cc-4b7d-a1cc-21beb45a3d88.png)
![image](https://user-images.githubusercontent.com/90484678/216175399-c762c42e-d474-4770-bc51-42d80f5db5f5.png)

**Reply by kazuohprc** [reply]
Eclipse uses a custom plug-in that supports Japanese and other languages.
I will upload an image showing the version of Eclipse.
![image](https://user-images.githubusercontent.com/90484678/216175778-18a6a246-c74a-4435-ace0-79ef43932be1.png)

project compiler
![image](https://user-images.githubusercontent.com/90484678/216175814-513a4439-72d2-4482-8275-72003a23c967.png)

project faset
![image](https://user-images.githubusercontent.com/90484678/216175939-543309e6-6f78-450a-a73a-5c8a27bfa747.png)

project build path
![image](https://user-images.githubusercontent.com/90484678/216175992-b46f587c-7df3-41bc-a5c4-e0edb30bcc20.png)

I couldn't describe the log because I don't know how to check for errors related to plugins.

**Reply by protoism** [reply]
@kazuohprc Can you file your results here https://github.com/gwt-plugins/gwt-eclipse-plugin/discussions/442?

**Comment by jetmore** [comment]
Very interested in providing testing support and possibly development support.  My boss and I sat down to discuss what felt like a crumbling GWT/gwt-eclipse-plugin after a lot of pain bringing up and environment for a new dev.  Switching away from GWT entirely is on the table but very painful obviously. We were very happy to find this attempt to rekindle the project.

* Currently we're on 2021-06 but some of the pain with the new dev environment has led us to experiment with 2022-12 and the version of the plugin from http://keeitsi.com/software/eclipse-plugins/
* We're a mac shop with versions of the operating system a bit all over the place. (I have 13.x and I believe I heard a coworker say they were on 11.x the other day).  We have both intel and m1 machines, and I think we might have an m2 machine also
* jdk11

**Reply by georgevaccaro** [reply]
@jetmore, I can't seem to get that link to work in 2022-12.  Any advice?
Also, I feel your pain.  I need to upgrade from my old machine to a new one and I was not prepared for gwt to be in such disrepair. :(

![Screenshot_2023-02-07_12-58-35](https://user-images.githubusercontent.com/2207417/217328047-abea8690-bb5b-4d2b-8fcc-ca47d43b8505.png)

**Reply by jetmore** [reply]
@georgevaccaro I don't know much, I just came in to my boss just saying "hey, I found this alternate version and it allowed [new dev's] enviroment work".  Here are the steps that got written up after that, maybe they will help but I can't provide much context
![Screenshot 2023-02-07 at 13 18 09](https://user-images.githubusercontent.com/978659/217331544-e6c54247-8772-48ec-97d2-92862094ca0a.png)

**Reply by jetmore** [reply]
Also, not to make them the support person for this, but in doing due diligence for installing random extension from the internet, I found what I believe to be the source for those changes in https://github.com/keinhaar/gwt-eclipse-plugin

**Reply by protoism** [reply]
Hi @jetmore, would you like to test the "official" plugin and add your results here: https://github.com/gwt-plugins/gwt-eclipse-plugin/discussions/442?
I'm very interested in MACOS results
Thanks!

**Reply by jetmore** [reply]
@protoism I and my employer are very interested but it likely won't be me for a little bit for Reasons(tm).  I'm going to tag a colleague into #442 so that we can still provide testing support.

**Reply by ishres19** [reply]
Hi @protoism, I am @jetmore's colleague and a new developer. I will try installing the official plugin, and will upload the results in #442, please let me know if there is any other way I can provide testing support. Thanks!

**Comment by keinhaar** [comment]
do we have an official beta version update site?
should be posted here and/or on the readme.md

**Comment by niloc132** [comment]
Test artifact is ready, marked 4.0.0. The repository download can be found at https://github.com/gwt-plugins/gwt-eclipse-plugin/suites/10728003175/artifacts/537867530 (the latest build on main, will be available for 90 days). 

Alternatively, you can download it from this update site, also temporarily until we have a release out: https://eclipse.gwtproject.org.vertispan.com/2023-02-01

**Reply by kazuohprc** [reply]
Thank you very much.
I have confirmed that the GWT SDK change bug has been fixed.
It was possible to change from GWT29 to GWT27, and the operation was good.

![image](https://user-images.githubusercontent.com/90484678/216491070-178cd599-53dd-4216-926b-d49431898fca.png)

**Comment by georgevaccaro** [comment]
> Hi all, does anyone want to candidate as a release tester? We need to try to reasonably cover the matrix

I can help test.
Version: 2022-12 (4.26.0)
Build id: 20221201-1913

Ubuntu 22.04.1 LTS

java --version=
openjdk 17.0.5 2022-10-18
OpenJDK Runtime Environment (build 17.0.5+8-Ubuntu-2ubuntu122.04)
OpenJDK 64-Bit Server VM (build 17.0.5+8-Ubuntu-2ubuntu122.04, mixed mode, sharing)

But I can switch to another jdk if needed.  This is a brand new machine with hardly anything installed on it.

**Reply by protoism** [reply]
@georgevaccaro can you post your results here, please? https://github.com/gwt-plugins/gwt-eclipse-plugin/discussions/442

**Reply by georgevaccaro** [reply]
Sure, just to be sure I'm doing the right thing, should I be adding a repository with this link: https://github.com/gwt-plugins/gwt-eclipse-plugin/suites/10728003175/artifacts/537867530 ?  And then I assume I find the GWT 4.0.0 plugin and install it, then try to compile a project etc.?  Is there an official test plan I can use so I don't waste anyone's time?

**Reply by protoism** [reply]
Have a look at https://github.com/gwt-plugins/gwt-eclipse-plugin/discussions/442, there are instructions step by step.
The repo at https://github.com/gwt-plugins/gwt-eclipse-plugin/suites/10728003175/artifacts/537867530 is really the same version, but zipped.
And Eclipse does not support zipped https update sites

**Comment by ngfchristian** [comment]
+1
Eclipse IDE for Entreprise Java and Web Developers V. 2022-09/Linux Ubuntu
18.04.6 /Java 17

On Tue, Jan 31, 2023 at 4:52 PM Luca Piccarreta ***@***.***>
wrote:

> Hi all,
> does anyone want to candidate as a release tester?
> We need to try to reasonably cover the matrix
>
>    - Eclipse version
>    - O/S
>    - Java version
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/gwt-plugins/gwt-eclipse-plugin/discussions/436>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADTO7QBIPFUG2Y2IICFSQB3WVE7LNANCNFSM6AAAAAAUMVQQPI>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
Christian Nzhie.

**Reply by niloc132** [reply]
Can you test using this discussion and post results there as well? https://github.com/gwt-plugins/gwt-eclipse-plugin/discussions/442
